website/docs/docs/core/connect-data-platform/cratedb-setup.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "CrateDB setup"
+description: "Read this guide to learn about the CrateDB warehouse setup in dbt."
+id: "cratedb-setup"
+meta:
+  maintained_by: Community
+  authors: 'CrateDB maintainers'
+  github_repo: 'crate/dbt-cratedb2'
+  pypi_package: 'dbt-cratedb2'
+  min_core_version: 'v1.0.0'
+  cloud_support: Supported
```

Comment:
Whether `cloud_support: Supported` is indeed true, we need to validate.

- https://github.com/crate/devrel/issues/58

Reply:
We learned from @hlcianfagna that this flag may only be toggled by people of dbt, because only popular and requested database plugins will gain "dbt Cloud support"? I am not sure about the semantics of this particular attribute yet.

Reply:
I've not changed it. Let's see what the outcome from the official review will be.